src/DetailsView/reports/components/report-sections/rules.tsx

```diff
@@ -0,0 +1,26 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+import * as React from 'react';
+
+import { NamedSFC } from '../../../../common/react/named-sfc';
+import { RuleResult } from '../../../../scanner/iruleresults';
+import { InstanceOutcomeType } from './outcome-summary-bar';
+import { RuleDetail, RuleDetailDeps } from './rule-detail';
+
+export type RulesDeps = RuleDetailDeps;
+
+export type RulesProps = {
+    deps: RulesDeps;
+    rules: RuleResult[];
+    outcomeType: InstanceOutcomeType;
+};
+
+export const Rules = NamedSFC<RulesProps>('RuleDetailsGroup', ({ rules, outcomeType, deps }) => {
+    return (
+        <div className="rule-details-group">
```

Comment:
Should this class name be different considering the name (and the fact we have 'RuleDetailsGroup' somewhere else as a component)

Reply:
Yes!!! that's part of the feedback I'm looking for. I don't have a good name for this component and `Rules` doesn't sound exactly right. I'll update the string name when I get to the final naming for the component itself.